src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFX.java

```diff
@@ -101,10 +101,10 @@ public void config() {
         turnEncoder.getConfigurator().apply(canCoderConfiguration);
 
         driveTalonFXConfiguration.Slot0 = new Slot0Configs()
-                .withKS(3.16)
-                .withKV(0.16023)
-                .withKA(3.5602)
-                .withKP(12.193);
+                .withKS(5.8432)
+                .withKV(0)
+                .withKA(2.4446)
+                .withKP(23.506);
```

Comment:
These are the new values from ~iffy tuning done at the practice field at Dalton. Needs a TODO to retune.